### ISSUE TITLE
Minimum viable burn-in integration

### ISSUE THREAD

**Issue by haikoschol** [paritytech/parity-processbot#306]
Supersedes #240, which was still overly complex and error-prone.

* developer adds the label `A1-needsburnin` on a PR
* If PR is in the substrate repo: processbot posts a comment to the PR which explains that burn-ins can currently only be done on polkadot PRs. Comment includes explaining the difference between companion and burn-in PR.
* If PR is in the polkadot repo: processbot posts a comment to the PR with a link to the burn-in frontend, mentioning that this is only accessible to Parity devs. Comment includes explaining the difference between companion and burn-in PR.

**Comment by mordamax** [comment]
@haikoschol could this be done with https://docs.github.com/en/actions/managing-issues-and-pull-requests/commenting-on-an-issue-when-a-label-is-added ?

**Comment by haikoschol** [comment]
> @haikoschol could this be done with https://docs.github.com/en/actions/managing-issues-and-pull-requests/commenting-on-an-issue-when-a-label-is-added ?

Sorry for the late response. I thought this ticket had already been closed.

The burn-in automation is currently unmaintained and no new features are being developed. We have a few alternative tools for testing such as [zombienet](https://github.com/paritytech/zombienet) and the self-serve tooling on testnets built by the devops team. I've moved to a different team a while ago and am working on something else now.
